### DISCUSSION TITLE
dataset

### DISCUSSION THREAD

**Discussion by zwsbh** [academicpages/academicpages.github.io#2243] [Q&A]
Hello, can I ask some questions about the AGRICULTURE VISION dataset

**Comment by zwsbh** [comment]
Can the dataset be used for one's own research

**Comment by rjzupkoii** [comment]
I'm not sure what dataset you are referring to?

**Reply by zwsbh** [reply]
Agriculture-Vision-2021

**Reply by zwsbh** [reply]
Is the subset of Agriculture Vision 2021 publicly researchable

**Reply by rjzupkoii** [reply]
@zwsbh That sounds like a project that's not related to the Academic Pages template, I haven't actually heard of that dataset in my own work.
